frontend/store/auctions.ts

```diff
@@ -161,13 +186,31 @@ export const actions = {
             });
             commit('setError', null);
             commit('setAuctions', auctions);
-        } catch (error) {
+        } catch (error: any) {
             console.error('fetch auction error', error);
             commit('setError', error.message);
         } finally {
+            commit('refreshLastUpdated');
             commit('setAreAuctionsFetching', false);
         }
     },
+    async updateSingleAuction({ commit, rootGetters }: ActionContext<State, State>, auctionId: string) {
+        const network = rootGetters['network/getMakerNetwork'];
+        if (!network) {
+            return;
+        }
+        commit('setIsSelectedAuctionFetching', true);
+        try {
+            const auction = await fetchSingleAuctionById(network, auctionId);
+            commit('setAuction', auction);
+            commit('setErrorByAuctionId', { auctionId, error: null });
```

Comment:
We should use null only if we need to distinguish from "no value" state (here and everywhere in our code). Here it's more appropriate to store the same value as the default one, ie `undefined`